sprout/core/create_resource_raw_path.py

```diff
@@ -0,0 +1,14 @@
+from pathlib import Path
+
+
+def create_resource_raw_path(path: Path) -> Path:
+    """Create a path to the raw directory for a resource.
+
+    Args:
+        path: Directory to create the new resource directory in.
```

Comment:
```suggestion
        path: Directory to create the new raw directory in.
```
Just because there's some ambiguity in "new resource directory" -- could be read as the directory for a specific resource and then its parent directory would be the resources folder.